testing/acceptance_test.py

```diff
@@ -969,6 +969,19 @@ def test(): pass
     assert r.ret == 0
 
 
+def test_pytest_plugins_as_module(testdir):
+    """Do not raise an error if pytest_plugins attribute is a module (#3899)"""
+    testdir.makepyfile(
+        **{
+            "utils/pytest_plugins.py": "",
+            "utils/__init__.py": "import pytest_plugins",
```

Comment:
this test is actually incomplete, as utils is not considered as pytest plugin 

you have to import it into a conftest

also use a a relative import

Reply:
You are right, done!

We don't even need subpackage, we just need `pytest_plugins` as module in a `conftest.py` file.